foxglove_bridge_base/include/foxglove_bridge/websocket_server.hpp

```diff
@@ -768,8 +770,10 @@ inline void Server<ServerConfiguration>::handleBinaryMessage(ConnHandle hdl, Mes
     case ClientBinaryOpcode::SERVICE_CALL_REQUEST: {
       ServiceRequest request;
       if (length < request.size()) {
-        sendStatusAndLogMsg(hdl, StatusLevel::Error,
-                            "Invalid service call request length " + std::to_string(length));
+        const std::string errMessage =
+          "Invalid service call request length " + std::to_string(length);
+        sendServiceFailure(hdl, request.serviceId, request.callId, errMessage);
+        sendStatusAndLogMsg(hdl, StatusLevel::Error, errMessage);
```

Comment:
Is the `sendStatusAndLogMsg()` redundant now?

Reply:
Well, a little bit, at least for recent clients that listen to both the log messages and the `serviceCallFailure` operation. I guess it's fine to remove it and instead to just log a message to console